src/ol3cesium.js

```diff
@@ -28,6 +28,14 @@ olcs.OLCesium = function(options) {
   this.map_ = options.map;
 
   /**
+   * @type {!Function}
+   * @private
+   */
+  this.time_ = options.time || function() {
```

Comment:
You need to add the time parameter to the olcsx.OLCesiumOptions typedef in olcsx.js.
The type should be `function(): Cesium.JulianDate`.